src/controllers/places.js

```diff
@@ -0,0 +1,17 @@
+const Request = require('request');
+
+const placeController = {};
+
+placeController.getAll = (req, res) => {
+  const placesURL = 'https://nazareth-open-tourism-platform.herokuapp.com/places';
+  Request(placesURL, (error, response, body) => {
+    if (error) return res.status(404).send(error);
+    const data = JSON.parse(body);
```

Comment:
I think we need to filter the data here to make sure we only have results that contain `en.name` fields (they don't all have to)